thirdeye/thirdeye-frontend/app/pods/components/detection-health/template.hbs

```diff
@@ -1,84 +1,103 @@
-            <li class="te-horizontal-metrics__number{{task.label}}">{{task.number}}</li>
-            <li class="te-horizontal-metrics__title">{{task.title}}</li>
-          </ul>
-        {{/each}}
-      </div>
+            {{#each tasks as |task|}}
+              <ul class="te-horizontal-metrics__metric">
+                <li class="te-horizontal-metrics__number{{task.label}}">{{task.number}}</li>
+                <li class="te-horizontal-metrics__title">{{task.title}}</li>
+              </ul>
+            {{/each}}
+          </div>
 
-      <label class="control-label te-label">
-        Anomaly coverage ratio
-      </label>
+          <label class="control-label te-label">
+            Anomaly coverage ratio
+          </label>
 
-      <div class="te-horizontal-metrics__container">
-        <ul class="te-horizontal-metrics__metric">
-          <li class="te-horizontal-metrics__number{{anomalyCoverage.label}}">{{anomalyCoverage.status}}</li>
-          <li class="te-horizontal-metrics__title">Status</li>
-        </ul>
+          <div class="te-horizontal-metrics__container">
+            <ul class="te-horizontal-metrics__metric">
+              <li class="te-horizontal-metrics__number{{anomalyCoverage.label}}">{{anomalyCoverage.status}}</li>
+              <li class="te-horizontal-metrics__title">Status</li>
+            </ul>
 
-        <ul class="te-horizontal-metrics__metric">
-          <li class="te-horizontal-metrics__number">{{anomalyCoverage.ratio}}%</li>
-          <li class="te-horizontal-metrics__title">The percentage of anomalous duration over the past 30 days</li>
-        </ul>
-      </div>
+            <ul class="te-horizontal-metrics__metric">
+              <li class="te-horizontal-metrics__number">{{anomalyCoverage.ratio}}%</li>
+              <li class="te-horizontal-metrics__title">The percentage of anomalous duration over the past 30 days</li>
+            </ul>
+          </div>
 
-      <p>To improve this, adjust alert sensitivity using preview.</p>
+          <p>To improve this, adjust alert sensitivity using preview.</p>
 
-      <label class="control-label te-label">
-        Regression error{{#if regressionInfo.rule}} for {{regressionInfo.rule}}{{/if}}
-      </label>
+          <label class="control-label te-label">
+            Regression error{{#if regressionInfo.rule}} for {{regressionInfo.rule}}{{/if}}
+          </label>
 
-      <div class="te-horizontal-metrics__container">
-        <ul class="te-horizontal-metrics__metric">
-          <li class="te-horizontal-metrics__number{{regressionInfo.label}}">{{regressionInfo.status}}</li>
-          <li class="te-horizontal-metrics__title">Status</li>
-        </ul>
+          <div class="te-horizontal-metrics__container">
+            <ul class="te-horizontal-metrics__metric">
+              <li class="te-horizontal-metrics__number{{regressionInfo.label}}">{{regressionInfo.status}}</li>
+              <li class="te-horizontal-metrics__title">Status</li>
```

Comment:
Some indent issue. it would be better to fix it.

Reply:
I am not sure that there is an issue. I have used spaces everywhere, don't see indentation issue in the editor. I think it is just showing highlighting in the diff the indentation has moved by 2 spaces so reviewers know what has changed, but that is expected as I have added a parent tag so the indentation change is expected.